### PR TITLE
Adding postgres 13

### PR DESCRIPTION
**Pull request by cliedeman**
Depends on #962

### CODE REVIEW

**Comment by a8m** [comment]
#962 merged, please rebase.

**Comment by cliedeman** [comment]
@a8m rebased

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -151,6 +151,18 @@ jobs:
...
+        env:
+          POSTGRES_DB: test
+          POSTGRES_PASSWORD: pass
+        ports:
+          - 5434:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
```
Comment (a8m) [inline .github/workflows/ci.yml:165]: Please add this to the migration workflow (line 260~) as well (GitHub doesn't support YAML anchors).

**Comment by a8m** [comment]
Thanks for your contribution @cliedeman 😃

**Pull request merged**